test/single/test_ray.py

```diff
@@ -28,7 +28,7 @@ def ray_start_2_cpus():
 
 @pytest.fixture
 def ray_start_4_cpus():
-    address_info = ray.init(num_cpus=4)
+    address_info = ray.init(num_cpus=4, _redis_max_memory=1024*1024*1024)
```

Comment:
does this fix some specific transient ray test errors?

Reply:
It fix the case in local. But the root cause of buildkite failure is the lightning version missmatch.

Reply:
I've also seen some random flakiness with ray in tests that requires changing these params. I think this is fine if it unblocks for now.